build-system/common/esbuild-babel.js

```diff
@@ -48,7 +48,7 @@ function getEsbuildBabelPlugin(
       .then((result) => {
         const {code, map} = result || {};
         debug('post-babel', filename, code, map);
-        return code + `\n// ${filename}`;
```

Comment:
This was so that I could grep the cache files for a filename. Please add it back, and let terser strip the comment.